<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactUserItem</class>
 <widget class="QDialog" name="ContactUserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>72</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QLabel" name="icon_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>45</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="red_point">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>2</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="user_name_label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
